### PR TITLE
dns: add missing dns keywords to schema.json v6

### PR DESCRIPTION
**Pull request by hadiqaalamdar**
Feature #5642

- [x] I have read the contributing guide lines at https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/issues/5642) ticket: https://redmine.openinfosecfoundation.org/issues/5642

Previous PR: https://github.com/OISF/suricata/pull/10432

Describe changes:
- made recommended changes from the last PR

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1588

### CODE REVIEW

File: etc/schema.json [hunk 1]
```diff
@@ -999,6 +999,10 @@
                     "description": "DNS opcode as an integer",
                     "type": "integer"
                 },
+                "tc": {
+                    "description": "A 1-bit subfield for truncated response that specifies if the length of the message exceeds the allowed length",
```
Comment (inashivb) [inline etc/schema.json:1003]: I'd like others' opinions on this but I think this looks very RFC definition and not very Suricata specific like what does this field in Suricata logs represent. Like `DNS truncated response field as a boolean`..
Note that this is not a change requested. Would like your and other opinions on this.

Comment (catenacyber) [reply etc/schema.json:1003]: I do not think there is a difference for this field between RFC and Suricata...

If I want to be nitpicky, I would say `1-bit subfield` is redundant

Comment (jufajardini) [reply etc/schema.json:1003]: Is your question towards description style? If so, I understand what you say. 
Have we ever discussed what we'd like these descriptions to look like? Any style guide or something?

Comment (jasonish) [reply etc/schema.json:1003]: I think it should be pretty simple, perhaps along the lines of:
`DNS truncation flag`.

We should actually say that this means the DNS truncation message was truncated.  Its easy to set this value to true or false whether the actual DNS message was truncated or not.  So this should just describe what that header value was set to.

**Comment by hadiqaalamdar** [comment]
New PR: https://github.com/OISF/suricata/pull/10476

**Pull request closed without merging**